plugins/activity/plugin.go

```diff
@@ -0,0 +1,91 @@
+package activity
+
+import (
+	"math/rand"
+	"sync"
+	"time"
+
+	"github.com/iotaledger/hive.go/daemon"
+	"github.com/iotaledger/hive.go/logger"
+	"github.com/iotaledger/hive.go/node"
+
+	"github.com/iotaledger/goshimmer/packages/shutdown"
+	"github.com/iotaledger/goshimmer/packages/tangle/payload"
+	"github.com/iotaledger/goshimmer/plugins/messagelayer"
+)
+
+const (
+	// PluginName is the plugin name of the activity plugin.
+	PluginName  = "Activity"
+	delayOffset = 10
+)
+
+var (
+	// plugin is the plugin instance of the activity plugin.
+	plugin *node.Plugin
+	once   sync.Once
+	log    *logger.Logger
+)
+
+// Plugin gets the plugin instance.
+func Plugin() *node.Plugin {
+	once.Do(func() {
+		plugin = node.NewPlugin(PluginName, node.Disabled, configure, run)
+	})
+	return plugin
+}
+
+// configure events
+func configure(_ *node.Plugin) {
+	log = logger.NewLogger(PluginName)
+
+	log.Infof("starting node with activity plugin")
```

Comment:
You can get rid of the logger as every plugin has an logger integrated. `plugin.LogInfo` or something should do it.